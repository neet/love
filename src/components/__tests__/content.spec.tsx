import * as React from 'react';
import * as ReactTestRenderer from 'react-test-renderer';
import Content from '../content';

describe('Content', () => {
  it('renders a content with given react node', () => {
    const tree = ReactTestRenderer.create(
      <Content>
        <p>This is a node</p>
      </Content>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
