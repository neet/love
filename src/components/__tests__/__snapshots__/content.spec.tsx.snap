// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders a content with given react node 1`] = `
<main
  className="content"
>
  <p>
    This is a node
  </p>
</main>
`;
